lib/os.js

```diff
@@ -54,3 +54,6 @@ if (binding.isBigEndian)
   exports.endianness = function() { return 'BE'; };
 else
   exports.endianness = function() { return 'LE'; };
+
+exports.sizeof = process.binding('sizeof');
+exports.alignof = process.binding('alignof');
```

Comment:
Wouldn't it be better to cache a return result here? (Or is that already what the c++ does?)